scripts/rename-schema.sh

```diff
@@ -9,8 +9,8 @@ perl -pi -e s,'<ExprRef>','',g build/vega-lite-schema.json
 perl -pi -e s,'\,ExprRef>','>',g build/vega-lite-schema.json
 
 perl -pi -e s,'CompositeEncoding','Encoding',g build/vega-lite-schema.json
-perl -pi -e s,'Generic(.*)<FacetedUnitSpec\,LayerSpec\,?.*\,FieldName>','\1',g build/vega-lite-schema.json
```

Comment:
Being normalized means it can only has FieldName, but not vice versa.  (normalized means much more.)  

I think you mean more like nonrepeated spec?

Reply:
Eventually, normalized specs shouldn't even appear in the schema and I'm working on that. In the meantime, this is flipping the naming to be more correct.